scripts/nginx/trackarr.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+. /etc/swizzin/sources/functions/utils
+user=$(_get_master_username)
+cat > /etc/nginx/apps/trackarr.conf << EOF
+location /trackarr/ {
+    proxy_pass              http://127.0.0.1:7337/trackarr/;
+    proxy_set_header        Host                    \$proxy_host;
+    proxy_set_header        X-Forwarded-Host        \$http_host;
+
+    proxy_set_header        Upgrade                 \$http_upgrade;
+    proxy_set_header        Connection              "Upgrade";
+    
+    auth_basic              "What's the password?";
+    auth_basic_user_file    /etc/htpasswd.d/htpasswd.${user};
+}
+EOF
+
+# Prevent double-basic-auth req if nginx was installed after trackarr
+sed -i '/user:/d' /opt/trackarr/config.yaml
+sed -i '/pass:/d' /opt/trackarr/config.yaml
```

Comment:
I don't think this is safe. Trackers like IPT accept a user/pass argument for SASL auth -- these sed lines could clobber other config options unintentionally

Reply:
Any suggestions how else could we achieve this? Just leave it unprotected without nginx?

Reply:
Also wouldn't that be two separate files? I'm not 100% sure as I don't run this myself yet

Reply:
No the config.yaml is all one file -- perhaps I am overthinking this. If swizzin just created the config file then there would be no reason for any other potentially conflicting config options to be present.

Reply:
> 
> 
> No the config.yaml is all one file -- perhaps I am overthinking this. If swizzin just created the config file then there would be no reason for any other potentially conflicting config options to be present.

unless nginx got installed after trackarr.

I can probably check for the specific section it's in or also use the actual credentials of the user so that something else doesn't get removed.

Reply:
I realised the credentials should not be there in the nginx case at all so no need to have that there lol